egui/src/widgets/plot.rs

```diff
@@ -300,24 +347,10 @@ impl Plot {
     /// Always fills the full height of the plot.
     pub fn vline(mut self, mut vline: VLine) -> Self {
         self.auto_color(&mut vline.stroke.color);
-        self = self.include_x(vline.x);
         self.vlines.push(vline);
         self
     }
 
-    /// Expand bounds to include the given x value.
-    pub fn include_x(mut self, x: impl Into<f64>) -> Self {
-        self.bounds.extend_with_x(x.into());
-        self
-    }
-
-    /// Expand bounds to include the given y value.
-    /// For instance, to always show the x axis, call `plot.include_y(0.0)`.
-    pub fn include_y(mut self, y: impl Into<f64>) -> Self {
-        self.bounds.extend_with_y(y.into());
-        self
-    }
-
```

Comment:
This is a feature I'd rather not lose. I think it makes sense if these would only apply to the default bounds, so that we would have `default_min_bounds: Bounds` as a member, which starts out `Bounds::EMPTY` and then is potentially expanded with `include_x` and `include_y`. The first frame (and on double-click) the automatic bounds would be computed from all curves in union with the `default_min_bounds`